kstreams/create.py

```diff
@@ -1,25 +1,28 @@
 from typing import Optional, Type
 
+from .backends.kafka import Kafka
 from .clients import Consumer, ConsumerType, Producer, ProducerType
 from .engine import StreamEngine
-from .prometheus.monitor import PrometheusMonitor, PrometheusMonitorType
+from .prometheus.monitor import PrometheusMonitor
 from .serializers import ValueDeserializer, ValueSerializer
 
 
 def create_engine(
     title: Optional[str] = None,
+    backend: Optional[Kafka] = None,
     consumer_class: Type[ConsumerType] = Consumer,
     producer_class: Type[ProducerType] = Producer,
     value_serializer: Optional[ValueSerializer] = None,
     value_deserializer: Optional[ValueDeserializer] = None,
-    monitor: Optional[PrometheusMonitorType] = None,
+    monitor: Optional[PrometheusMonitor] = None,
 ) -> StreamEngine:
 
     if monitor is None:
         monitor = PrometheusMonitor()
 
```

Comment:
I think the `engine` can not exist without a backend`, maybe we should add here: 

```python
if backend is None:
    backend = Kafka()
```